docs-chef-io/content/inspec/cli.md

```diff
@@ -44,50 +39,59 @@ This subcommand has the following additional options:
 * `--tar`, `--no-tar`
     Generates a tar.gz archive.
 * `--vendor-cache=VENDOR_CACHE`
-    Use the given path for caching dependencies, (default: `~/.inspec/cache`).
+    Use the given path for caching dependencies, (default: ~/.inspec/cache).
 * `--zip`, `--no-zip`
     Generates a zip archive.
 
-## automate
+## check
 
-Communicates with Chef Automate.
+Verify the metadata in the `inspec.yml` file,  verify that control blocks have the correct fields (title, description, impact),  and define that all controls have visible tests and the controls are not using deprecated inspec dsl code
 
 ### Syntax
 
 This subcommand has the following syntax:
 
 ```bash
-inspec automate SUBCOMMAND
+inspec check PATH
 ```
 
-## check
+### Options
 
-Verify the metadata in the `inspec.yml` file, verify that control blocks have the correct fields (title, description, impact), and define that all controls have visible tests and the controls are not using deprecated InSpec DSL code.
+This subcommand has the following additional options:
+
+* `--auto-install-gems`, `--no-auto-install-gems`
+    Auto installs gem dependencies of the profile or resource pack.
+* `--format=FORMAT`
+    The output format to use. Valid values: `json` and `doc`. Default value: `doc`.
```

Comment:
Is there any chance we could change the format of the options so they're a [description list.](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl) It would look like:

```
`--auto-install-gems`, `--no-auto-install-gems`
: Auto installs gem dependencies of the profile or resource pack.

`--format=FORMAT`
: The output format to use. Valid values: `json` and `doc`. Default value: `doc`.
```

or: 

```
flag
: Flag description
```

Which would render like the options here: https://docs.chef.io/workstation/chef_shell/#options

Reply:
The below way is unable to indent, have used `&nbsp;` to force indent the description
```
flag
: Flag description
```

Reply:
Could you give a try to the current cli.md? @IanMadd

Reply:
Now it looks like this:
![Screenshot 2023-01-31 at 10-06-03 InSpec CLI](https://user-images.githubusercontent.com/8138169/215797557-a2709081-d229-42a2-94f5-562ad37ad857.png)


I updated the Netlify settings so it will build the site against this PR. By default it only builds previews to PRs against the main branch. That should help.

Reply:
Used the html tags to format the data, that seems to be working fine. @IanMadd

Reply:
That looks much better, but is there any way to wrap a group of options together in one <dl>. Right now it's producing a separate <dl> for each option. Everything else looks great though.

Reply:
It's done! @IanMadd